src/OpenLoco/src/Windows/MapWindow.cpp

```diff
@@ -31,6 +31,8 @@
 #include "Ui/LastMapWindowAttributes.h"
 #include "Ui/ScrollView.h"
 #include "Ui/Widget.h"
+#include "Ui/Widgets/FrameWidget.h"
+#include "Ui/Widgets/PanelWidget.h"
```

Comment:
Is the plan really to include all widget types used in every unit file? Particular widgets are going to be used by pretty all windows, after all. I think there's logic in including frame, panel, (close) button by default at least.

Reply:
Not sure I like that, there might be smaller windows where 90% of the widgets are unused then, I think its best to just pick the widgets required otherwise pointless code crunching for the unit.

Reply:
I did mean frame/panel/button as an exhaustive list. Indeed, no point in including _all_ widget code by default.